oauth/googleoauth/config.go

```diff
@@ -0,0 +1,21 @@
+package googleoauth
+
+import (
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+
+	"github.com/Shopify/goose/oauth"
+)
+
+func NewConfig(clientID string, clientSecret string, paths *oauth.Paths) *oauth2.Config {
+	return &oauth2.Config{
+		ClientID:     clientID,
+		ClientSecret: clientSecret,
+		RedirectURL:  paths.CallbackURL().String(),
+		Scopes: []string{
```

Comment:
I guess the OpenID Connect scopes `email` and `profile` should be sufficient? (Only userinfo.email won't give us the name, right?)

Reply:
I thought about it, but felt this would serve most purposes. One can always build their own `oauth2.Config` or override the value afterwards. I don't have a strong preference 🤷‍♂️